.github/workflows/main.yml

```diff
@@ -176,6 +176,39 @@ jobs:
       #      -Ddep.iceberg.version=${ICEBERG_VERSION} \
       #      -Ddep.nessie.version=${NESSIE_VERSION}
 
+  icberg10_nessie043:
+    name: Nessie 0.43 / Iceberg 1.0
+    runs-on: ubuntu-latest
+    env:
+      SPARK_LOCAL_IP: localhost
+
+    steps:
+      - name: Checkout Integrations Testing repo
+        uses: actions/checkout@v3
+
+      - name: Setup gradle.properties
+        run: |
+          mkdir -p ~/.gradle
+          echo "org.gradle.jvmargs=-Xmx1024m -XX:MaxMetaspaceSize=768m -Dfile.encoding=UTF-8" >> ~/.gradle/gradle.properties
+          echo "org.gradle.vfs.watch=false" >> ~/.gradle/gradle.properties
+
+      - name: Set up JDK ${{ matrix.java-version }}
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
```

Comment:
didnt we go with temurin in another repo since they come pre-cached? maybe we can do it in a followup